### PR TITLE
KOGITO-4440 Quarkus 1.12.0.Final

### PR DESCRIPTION
**Pull request by evacchi**
https://issues.redhat.com/browse/KOGITO-4440

assembly:
- https://github.com/kiegroup/kogito-runtimes/pull/1069
- https://github.com/kiegroup/kogito-apps/pull/647
- https://github.com/kiegroup/kogito-examples/pull/577

This _does_ not include the Infinispan upgrade.

### CODE REVIEW

**Comment by evacchi** [comment]
@tiagodolphine it looks like vert.x changed some imports and it affects jobs service; can you take a look there?

```
    import io.vertx.axle.core.eventbus.EventBus;
```

to:

```
    import io.vertx.mutiny.core.eventbus.EventBus;

```

(I think)  but this line won't compile

```
        Publisher<ObjectNode> publisher = messageConsumer.toPublisherBuilder().map(Message::body).buildRs();
```

**Comment by evacchi** [comment]
Jenkins please retest this

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
did you try the LTS check ? :D

**Comment by evacchi** [comment]
not yet @radtriste 
also now it's OOM'img 🤦

**Comment by evacchi** [comment]
Jenkins please retest this

File: kogito-build-parent/pom.xml [hunk 1]
```diff
@@ -722,12 +728,6 @@
         <version>${version.io.smallrye.reactive}</version>
       </dependency>
 
-      <dependency>
```
Comment (cristianonicolai) [inline kogito-build-parent/pom.xml:725]: @evacchi dep management not working? :/

Comment (evacchi) [reply kogito-build-parent/pom.xml:725]: I am removing that from BOM otherwise it's overriding quarkus' in apps

**Review by tiagodolphine — APPROVED** [review]

**Comment by radtriste** [comment]
Jenkins retest this

**Comment by evacchi** [comment]
Jenkins retest this

**Comment by evacchi** [comment]
Jenkins retest this

**Comment by evacchi** [comment]
Jenkins retest this

**Pull request merged**